api/snakeyaml-engine-kmp.api

```diff
@@ -48,7 +47,64 @@ public final class it/krzeminski/snakeyaml/engine/kmp/api/DumpSettings {
 	public final fun isUseUnicodeEncoding ()Z
 }
 
-public abstract interface class it/krzeminski/snakeyaml/engine/kmp/api/DumpSettings$MutableDumpSettings {
+public final class it/krzeminski/snakeyaml/engine/kmp/api/DumpSettingsCopyDslKt {
```

Comment:
The API changed (formally) because now we have a top-level interface and an extension function instead of both being inside the annotated classes. Its required because KSP cannot modify existing classes, it can only produce new entities. Since we haven't released the library since the last big breaking change, there's no need to account for this technically breaking change. The intended use of the the `copy` function remains unchanged.

Reply:
See:

> Its required because KSP cannot modify existing classes, it can only produce new entities.

I don't know if it's possible to work around it in a sensible way. I think IDE will still suggest what extension functions are available for a given type, however I cannot check right now.

Reply:
Oh, compatibility with Java - yeah, I'd actually assume we don't need to care about it that much since it's a Kotlin-centric library, as opposed to snakeyaml-engine.